distributor/cmd/internal/distributor/distributor_test.go

```diff
@@ -667,6 +667,137 @@ func TestGetCheckpointN(t *testing.T) {
 	}
 }
 
+func TestGetCheckpointNHistoric(t *testing.T) {
+	ws := map[string]note.Verifier{
+		"Whittle": witWhittle.verifier,
```

Comment:
May I respectfully request 3 witness/variable names that are quite a bit more visually distinct? :)

Somewhat embarrassingly, it took me a moment to notice in the test data structs that there were 3 witnesses, rather than 2 with one sending an updated checkpoint :)

Reply:
Should have gone to SpecSavers :-)

Changed to animal names.